### PR TITLE
Implement python emitters for padded ignore/content/strict/layout regions

### PR DESCRIPTION
**Pull request by ikseek**
Transforms template call
```
eyes.check({
      isFully: true,
      ignoreRegions: [{region: '#ignoreRegions', padding: 20}],
      layoutRegions: [{region: '#layoutRegions', padding: {top: 20, right: 20}}],
      contentRegions: [{region: '#contentRegions', padding: {right: 20, left: 20}}],
      strictRegions: [{region: '#strictRegions', padding: {bottom: 20} }]
    })
```
to
```
Target.window().ignore("#ignoreRegions", padding=20).layout("#layoutRegions", padding={"top":20,"right":20}).content("#contentRegions", padding={"right":20,"left":20}).strict("#strictRegions", padding={"bottom":20}).fully(True)
```

### CODE REVIEW

File: python/parser.js [hunk 1]
```diff
@@ -93,11 +95,8 @@ function region(region_param, first_call) {
     }
 }
 
-function ignoreRegions(arr) {
-    return arr.reduce((acc, val) => `${acc}.ignore(${regionParameter(val)})`, '')
-}
-function layoutRegions(arr){
-    return arr.reduce((acc, val) => `${acc}.layout(${regionParameter(val)})`, '')
+function regions(kind, arr) {
+    return arr.reduce((acc, val) => `${acc}.${kind}(${regionParameter(val)})`, '')
```
Comment (Yarden-Ingber) [inline python/parser.js:99]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
    return arr.reduce((acc, val) => `${acc}.${kind}(${regionParameter(val)})`, "")
```

File: python/parser.js [hunk 2]
```diff
@@ -18,10 +18,12 @@
         if (cs.frames) element += frames(cs.frames)
         if (cs.region) element += region(cs.region, true)
     }
-    if (cs.ignoreRegions) options += ignoreRegions(cs.ignoreRegions)
     if (cs.floatingRegions) options += floatingRegions(cs.floatingRegions)
     if (cs.accessibilityRegions) options += accessibilityRegions(cs.accessibilityRegions)
-    if (cs.layoutRegions) options += layoutRegions(cs.layoutRegions)
+    if (cs.ignoreRegions) options += regions("ignore", cs.ignoreRegions)
```
Comment (Yarden-Ingber) [inline python/parser.js:23]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
    if (cs.ignoreRegions) {options += regions("ignore", cs.ignoreRegions)}
```

File: python/parser.js [hunk 3]
```diff
@@ -136,7 +135,13 @@
             string = `${JSON.stringify(region)}`
             break;
         case "object":
-            string = parseObject(region.type ? region : (region.shadow ? region : {value: region, type:'Region'}))
+            if (region.region) {
+                string = python`${region.region}, padding=${region.padding}`
+            } else {
+                string = parseObject(
+                    region.type ? region : (region.shadow ? region : {value: region, type:'Region'})
```
Comment (Yarden-Ingber) [inline python/parser.js:142]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
                    region.type ? region : (region.shadow ? region : {value: region, type:"Region"})
```

File: python/parser.js [hunk 4]
```diff
@@ -18,10 +18,12 @@ function checkSettings(cs) {
         if (cs.frames) element += frames(cs.frames)
         if (cs.region) element += region(cs.region, true)
     }
-    if (cs.ignoreRegions) options += ignoreRegions(cs.ignoreRegions)
     if (cs.floatingRegions) options += floatingRegions(cs.floatingRegions)
     if (cs.accessibilityRegions) options += accessibilityRegions(cs.accessibilityRegions)
-    if (cs.layoutRegions) options += layoutRegions(cs.layoutRegions)
+    if (cs.ignoreRegions) {options += regions("ignore", cs.ignoreRegions)}
```
Comment (Yarden-Ingber) [inline python/parser.js:23]: Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
    if (cs.ignoreRegions) {options += regions("ignore", cs.ignoreRegions);}
```

File: python/parser.js [hunk 5]
```diff
@@ -18,10 +18,12 @@
...
+    if (cs.layoutRegions) {options += regions("layout", cs.layoutRegions)}
+    if (cs.contentRegions) {options += regions("content", cs.contentRegions)}
```
Comment (Yarden-Ingber) [inline python/parser.js:25]: Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
    if (cs.contentRegions) {options += regions("content", cs.contentRegions);}
```

File: python/parser.js [hunk 6]
```diff
@@ -18,10 +18,12 @@
...
+    if (cs.strictRegions) {options += regions("strict", cs.strictRegions)}
```
Comment (Yarden-Ingber) [inline python/parser.js:26]: Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
    if (cs.strictRegions) {options += regions("strict", cs.strictRegions);}
```

File: python/parser.js (hunk 5 above)
Comment (Yarden-Ingber) [inline python/parser.js:25]: Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
    if (cs.contentRegions) {options += regions("content", cs.contentRegions);}
```

File: python/parser.js (hunk 6 above)
Comment (Yarden-Ingber) [inline python/parser.js:26]: Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
    if (cs.strictRegions) {options += regions("strict", cs.strictRegions);}
```

File: python/parser.js [hunk 7]
```diff
@@ -18,10 +18,12 @@
         if (cs.frames) element += frames(cs.frames)
         if (cs.region) element += region(cs.region, true)
     }
-    if (cs.ignoreRegions) options += ignoreRegions(cs.ignoreRegions)
     if (cs.floatingRegions) options += floatingRegions(cs.floatingRegions)
     if (cs.accessibilityRegions) options += accessibilityRegions(cs.accessibilityRegions)
-    if (cs.layoutRegions) options += layoutRegions(cs.layoutRegions)
+    if (cs.ignoreRegions) {options += regions("ignore", cs.ignoreRegions)}
```
Comment (Yarden-Ingber) [inline python/parser.js:23]: Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
    if (cs.ignoreRegions) {options += regions("ignore", cs.ignoreRegions);}
```

**Review by Artem-applitools — APPROVED** [review]

**Pull request merged**